gadget/install/install.go

```diff
@@ -64,6 +64,9 @@ func Run(gadgetRoot, device string, options Options, observer gadget.ContentObse
 	if err != nil {
 		return fmt.Errorf("cannot layout the volume: %v", err)
 	}
+	if err := gadget.ResolveContentPaths(gadgetRoot, kernelRoot, lv); err != nil {
```

Comment:
This should not be done here but wherever we validate snap/gadget.yaml, should validate this as well. Namely, the path must be a correct resource in a gadget container or a valid expansion (using known variables and no ../.. things) of a variable.

Reply:
We can only ResolveContentPaths once we know what gadget and kernel are combined. So yes, validating earlier in addition is a good idea.

We need to use this also in gadget.Update() but I think those are the only places right now.

Reply:
Fwiw, we also will to "ResolveContentPaths()" in prepare-image during image building so any errors/mismatches will be detected early.

Reply:
Thinking further I think having this here at this point is premature so I remove it for now.